runtime/browser/runtime.cc

```diff
@@ -27,39 +30,88 @@
 #include "common/logger.h"
 #include "common/profiler.h"
 #include "runtime/browser/native_app_window.h"
-#include "runtime/common/constants.h"
+#include "runtime/browser/notification_window.h"
 #include "runtime/browser/preload_manager.h"
+#include "runtime/common/constants.h"
 
 namespace runtime {
 
 namespace {
 
-static NativeWindow* CreateNativeWindow() {
+const char kCategoryAlwaysOnTop[] = "http://tizen.org/category/always_on_top";
+
+static NativeWindow* CreateNativeWindow(NativeWindow::Type windowType) {
   SCOPE_PROFILE();
-  // TODO(wy80.choi) : consider other type of native window.
-  auto cached = PreloadManager::GetInstance()->GetCachedNativeWindow();
-  NativeWindow* window = cached != nullptr ? cached : new NativeAppWindow();
+
+  NativeWindow* window;
+
+  switch (windowType) {
+    case NativeWindow::Type::NORMAL: {
+      auto cached = PreloadManager::GetInstance()->GetCachedNativeWindow();
+      window = cached != nullptr ? cached : new NativeAppWindow();
+    }
+    break;
+    case NativeWindow::Type::NOTIFICATION: {
+      PreloadManager::GetInstance()->ReleaseCachedNativeWindow();
+      NotificationWindow* win = new NotificationWindow();
+      window = win;
+    }
+    break;
+  }
+
   window->Initialize();
   return window;
 }
 
+bool HasAlwaysOnTopCategory(
```

Comment:
Category of application is Permission
appcontrol data about alway_on_top is command to create notification window

Your implementation exchanged each others